locales/en.json

```diff
@@ -119,7 +119,7 @@
     "delete_list": {
       "cancel": "Cancel",
       "confirm": "Delete",
-      "title": "Are you sure you want to delete this list?"
+      "title": "Are you sure you want to delete \"{0}\" list?"
```

Comment:
```suggestion
      "title": "Are you sure you want to delete the \"{0}\" list?"
```

Reply:
Ups, sorry, please send a new PR 🙏🏼

Reply:
@boehs send a PR to fix it please :pray: , can you include that change in your current PR?